src/pages/Detail/StoreReviewBox/ReviewInput/ReviewInput.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import './ReviewInput.scss';
+
+class ReviewInput extends React.Component {
+  state = {
+    isTextValid: false,
+    isRatingValid: false,
+    reviewRating: [false, false, false, false, false],
+  };
+
+  handleReviewInput = e => {
+    e.target.value.length > 9
+      ? this.setState({ isTextValid: true })
+      : this.setState({ isTextValid: false });
+  };
```

Comment:
```suggestion
  handleReviewInput = e => {
    const isTextValid = e.target.value.length > 9
    
    this.setState({ isTextValid })
  };
```
- this.setState를 두번 작성하기 보다 isTextValid의 값을 구성해서 한번에 넘겨주는게 좋겠네요!